### PR TITLE
Fix: Correcting sections

### PR DESCRIPTION
**Pull request by SpeedyD**
Three pages in home and one in the admin section were not properly placed. This fixes it.

(It's the four pages left unchanged in #850.)

### CODE REVIEW

**Review by AW0005 — APPROVED** [review]

File: resources/views/home/bug_report_index.blade.php [hunk 1]
```diff
@@ -1,10 +1,10 @@
-@extends('layouts.app')
+@extends('home.layout')
```
Comment (itinerare) [inline resources/views/home/bug_report_index.blade.php:1]: Mind that the bug report index itself does not require being logged in to view-- so it *not* using the home layout is arguably more sensible.

**Review by itinerare — CHANGES_REQUESTED** [review]

File: resources/views/home/bug_report_index.blade.php (hunk 1 above)
Comment (SpeedyD) [reply resources/views/home/bug_report_index.blade.php:1]: Hrm.. I suppose that is true.. Perhaps it should be moved altogether?

Comment (itinerare) [reply resources/views/home/bug_report_index.blade.php:1]: That would probably be the better call.

Comment (SpeedyD) [reply resources/views/home/bug_report_index.blade.php]: Never mind. Undone this change.

**Review by itinerare — APPROVED** [review]

**Pull request merged**